### ISSUE TITLE
sticky navbar example not in tab order

### ISSUE THREAD

**Issue by aleavikraman** [themesberg/flowbite#619]
**Describe the bug**
The sticky navbar example has a WCAG 2.1 accessibility violation where the tab order of nav items is not in logical order. It tabs from the logo to the "get started" button and then back to the center nav items. The order should go from left to right in order to provide a better experience for keyboard only users

**To Reproduce**
1. Go to sticky navbar component
2. use tab key across all nav items
3. tab focus goes from logo to "get started" button

**Expected behavior**
tab order goes left to right across all nav items

**Screenshots**

**Desktop (please complete the following information):**
 - Browser chrome

**Smartphone (please complete the following information):**

**Additional context**
I wasn't sure where exactly to put this. It's an accessibility bug, but in reality it's a documentation issue with your examples. If this is the wrong area please advice me on where to create tickets like this in the future. Thanks
